enterprise/payara.common/nbproject/org-netbeans-modules-payara-common.sig

```diff
@@ -503,7 +503,7 @@ meth public final void setName(java.lang.String)
 meth public final void setPriority(int)
 meth public final void stop()
  anno 0 java.lang.Deprecated()
-meth public final void stop(java.lang.Throwable)
+meth public final void stop(java.lang.Throwable)	
```

Comment:
Is this an unnecessary change just adding empty space at the end of the line?

Reply:
Fixed the formatting issue.